docs/UserGuide.md

```diff
@@ -880,18 +930,18 @@ Some commands require you to include parameters. These parameters are identified
-| [s/]   | Score                      | 7 + 2 decimals          | For Exam Max Scores: the input must be a positive integer <br/> For Persons' Exam Scores: the input must be an integer greater than or equal to zero                                                                                                                                                                                                                                                                                                                                                                             |
-| [t/]…  | Tags                       | 100                     | Should be alphanumeric, and should not contain spaces.                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
-| \{lt\|\} | Less Than                | 7 + 2 decimals | Should be a positive numerical value smaller than the currently selected Exam's Max Score                                                                                                                                                                                                                                                                                                                                                                                                                                        |
-| \{mt\|\} | More Than                | 7 + 2 decimals | Should be a positive numerical value smaller than the currently selected Exam's Max Score                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| Prefix | What it refers to          | Max. Length | Constraints    |
+|--------|-----------------------------|-------------|----------------|
```

Comment:
Glad this was rectified. In future we need to ensure merges do not revert changes made.